data/low_income_thresholds.json

```diff
@@ -154,6 +154,48 @@
       }
     }
   },
+  "DC": {
+    "dc-dc-sustainable-energy-utility": {
+      "source_url": "https://www.dcseu.com/affordable-home-electrification",
+      "thresholds": {
+        "1": 85200,
+        "2": 97350,
+        "3": 109500,
+        "4": 121700,
+        "5": 133850,
+        "6": 146000,
+        "7": 158200,
+        "8": 170350
+      }
+    },
+    "dc-dc-department-of-energy-and-environment": {
+      "/": "DOEE uses 60% AMI as the threshold for their Weatherization Assistance Program, but 80% for Solar for All. I believe that in cases like this, we opt for the higher threshold values. See https://doee.dc.gov/service/wap for 60% AMI values",
```

Comment:
The right thing to do here is to define both sets of thresholds, and assign them to the appropriate records/rows. The spreadsheet-to-json script is not going to help you there, unfortunately, since it assumes that the authority is the name of the low-income threshold. You'll have to do that manually after the script runs (and unfortunately that will also prevent the sync test from being turned on for DC).

This is a known issue/gap with our low-income approahch in general, but I'm planning on addressing it next week (https://app.asana.com/0/0/1206728986903186/f).